src/CheckView.cpp

```diff
@@ -294,6 +294,8 @@ CheckView::HandleNotify(const uint64& value, const BMessage* msg)
 				}
 			}
 		}
+	} else if (value & WATCH_CHANGE) {
```

Comment:
Not sure how strict or if you apply the Haiku coding style in this project.
The curly brackets aren't needed for the single-line statement.

Otherwise the change looks good!

Reply:
I'll remove them. I also forgot to generate the catkeys file... Will update the PR in a minute

Reply:
@humdingerb `git-haiku-format` added the curly brackets back, so I think I'll just leave them there.
I generated the catkeys, and also found a few other cases where closed accounts should be taken into account (editing account settings and transfer between accounts).

Reply:
Weird. I thought haiku-format did respect hat part of the coding style... No matter.